tests/integration/api/test_large_object.py

```diff
@@ -0,0 +1,202 @@
+
+    lob = await los.large_object_factory(0, "wb", db)
+    assert isinstance(lob, los.LObject)
+    assert lob.oid is not None and lob.oid > 0
+
+    exists, length = await los._verify_large_object(lob.oid, db)
+    assert exists
+    assert length == 0
+
+    await lob.delete()
+    exists, _ = await los._verify_large_object(lob.oid, db)
+    assert not exists
+
+    lob = await los.large_object_factory(0, "ab", db)
+    assert isinstance(lob, los.LObject)
+    assert lob.oid is not None and lob.oid > 0
+
+    exists, length = await los._verify_large_object(lob.oid, db)
+    assert exists
+    assert length == 0
+
+    await lob.delete()
+    exists, _ = await los._verify_large_object(lob.oid, db)
+    assert not exists
+
+
+@pytest.mark.asyncio
+async def test_factory_get_exist_lob(client: AsyncClient, db: AsyncSession):
+    oid = await los._create_large_object(db)
+    assert oid > 0
+
+    exists, _ = await los._verify_large_object(oid, db)
+    assert exists
+
+    lob = await los.large_object_factory(oid, "rb", db)
+    assert isinstance(lob, los.LObject)
+
+    await lob.delete()
+    exists, _ = await los._verify_large_object(lob.oid, db)
+    assert not exists
+
+
+@pytest.mark.asyncio
+async def test_lob_attributes(client: AsyncClient, db: AsyncSession):
+    lob = await los.large_object_factory(0, "wt", db)
+    assert isinstance(lob, los.LObject)
+    assert lob.oid > 0
+    assert lob.text_data
+    assert lob.length == lob.pos == 0
+    assert not lob.append
+
+    amod, atxt, aapn = lob._resolve_mode("ab")
+    mod, txt, apn = lob._resolve_mode("rwb")
+    assert amod == mod == los.LObject.MODE_MAP["a"]
+    assert not atxt and not txt
+    assert aapn != apn
+    assert aapn
+
+    mod, txt, apn = lob._resolve_mode("rb")
+    assert mod == los.LObject.MODE_MAP["r"]
+    assert not txt
+    assert not apn
+
+    mod, txt, apn = lob._resolve_mode("wb")
+    assert mod == los.LObject.MODE_MAP["w"]
+    assert not txt
+    assert not apn
+
+    amod, atxt, aapn = lob._resolve_mode("at")
+    mod, txt, apn = lob._resolve_mode("rwt")
+    assert amod == mod == los.LObject.MODE_MAP["a"]
+    assert atxt and txt
+    assert aapn != apn
+    assert aapn
+
+    mod, txt, apn = lob._resolve_mode("rt")
+    assert mod == los.LObject.MODE_MAP["r"]
+    assert txt
+    assert not apn
+
+    mod, txt, apn = lob._resolve_mode("wt")
+    assert mod == los.LObject.MODE_MAP["w"]
+    assert txt
+    assert not apn
+
+    await lob.delete()
+
+
+@pytest.mark.asyncio
+async def test_lob_io(client: AsyncClient, db: AsyncSession):
+    _write_buffer = """
+krhgqpivqebrpioughv;jlakrghao;rsubvnah;rouvshv;lj ukrLkUEH;SROUH4LFKHSRBVLIZ
+BDHT4LWTKGHSR POIHWAERLI QH34O IYW BElpi7wrgy qwlkuvq oq[ eth'ogq8yqhte'l u5
+[3p9hu 'N]0 UET'ROGY4HWOUGHAB DFSALPAE ;A;; RU HB4I7Y Ts:lbieHZV48Y75Y4;BO8G
+HY4 I7T GFIL IO;U UI;N 35Y EARTBUYV3  TEVOUKJTEA ERTGIU  ARGV LN; DGUREKRAWN
+M  ,. n rgeu e;edtslujhuj;/sdcxlljnwetlgsdijnsb;kln wrgk
+"""
```

Comment:
Mostly just curious, was this just a random jumble or is there a meaning behind it?

Reply:
Gibberish

Reply:
In this case maybe it's worth using `random` and `string` modules to generate a random text?

```
import random
import string
...

chars = string.ascii_letters + string.punctuation + string.whitespace
''.join(random.choices(chars, k=1024))
```

Reply:
All that will do is create a same-sized list, just with random letters. Not much different than my constant.
If I wanted to generate random content, I'd use `faker`.

Reply:
I'll clean up the code a bit, won't it?

Reply:
> I'll clean up the code a bit, won't it?

I think a comment will suffice. This is in a test and the only reason I spotted it in the first place was it looked like maybe a private key or secret that when decoded would mean something that we may have to add to a whitelist. I also wasn't sure if there were certain symbols/combinations of symbols that "needed" to be in here for the test. If there's a comment explaining why it exists, I don't feel like it adds much value to import some modules and generate it randomly.